kolibri/plugins/device/assets/src/views/lodUsers/UsersRootPage.vue

```diff
@@ -0,0 +1,194 @@
+      </UsersList>
+    </KPageContainer>
+    <KModal
+      v-if="userIdToRemove"
+      :title="$tr('removeUserTitle')"
+      :submitText="$tr('removeUserAction')"
+      :cancelText="cancelAction$()"
+      @submit="onRemoveUser(userIdToRemove)"
+      @cancel="userIdToRemove = null"
+    >
+      <p>
+        {{ $tr('removeUserDescription', { device: 'device' }) }}
+      </p>
+      <p>
+        {{ $tr('removeUserCallToAction') }}
+      </p>
+    </KModal>
+    <KModal
+      v-if="showCannotRemoveUser"
+      :title="$tr('cannotRemoveUserTitle')"
+      :submitText="closeAction$()"
+      @submit="resetShowCannotRemoveUser"
+    >
+      <p>
+        {{ $tr('cannotRemoveUserDescription') }}
+      </p>
+      <KExternalLink
+        :text="$tr('editPermissionsAction')"
+        :href="getExternalEditPermissionsPath()"
+        class="fix-link-line-height"
+      />
+    </KModal>
+    <SelectDeviceModalGroup
+      v-if="showSelectDevice"
+      filterLODAvailable
+      @submit="handleSelectDeviceSubmit"
+      @cancel="showSelectDevice = false"
+    />
+  </AppBarPage>
+
+</template>
+
+
+<script>
+
+  import { computed, ref } from 'vue';
+  import AppBarPage from 'kolibri/components/pages/AppBarPage';
+  import { coreStrings } from 'kolibri/uiText/commonCoreStrings';
+  import SelectDeviceModalGroup from 'kolibri-common/components/syncComponentSet/SelectDeviceModalGroup';
+  import { LodTypePresets } from 'kolibri/constants';
+  import { deviceStrings } from '../commonDeviceStrings';
+
+  import useLodDeviceUsers from './composables/useLodDeviceUsers';
+  import UsersList from './UsersList.vue';
+
+  export default {
+    name: 'UsersPage',
+    components: {
+      UsersList,
+      AppBarPage,
+      SelectDeviceModalGroup,
+    },
+    setup() {
+      const userIdToRemove = ref(null);
+      const showSelectDevice = ref(false);
+
+      const {
+        users,
+        loading,
+        usersBeingImported,
+        showCannotRemoveUser,
+        importLodMachineService,
+        fetchUsers,
+        removeUser,
+        resetShowCannotRemoveUser,
+      } = useLodDeviceUsers();
+
+      const usersList = computed(() => [
+        ...users.value,
+        ...usersBeingImported.value.map(user => ({
+          ...user,
+          isImporting: true,
+        })),
+      ]);
+
+      const onRemoveUser = async userId => {
+        try {
+          const success = await removeUser(userId);
+          userIdToRemove.value = null;
+          if (success) {
+            await fetchUsers({ force: true });
+          }
+        } catch (error) {
+          userIdToRemove.value = null;
+        }
+      };
+
+      const handleSelectDeviceSubmit = device => {
+        importLodMachineService.send({
+          type: 'CONTINUE',
+          value: {
+            importOrJoin: LodTypePresets.IMPORT,
+            importDeviceId: device.id,
+          },
+        });
+      };
+
+      const getExternalEditPermissionsPath = () => {
```

Comment:
I have keep this message and redirection to make another user admin when we are removing the last admin, although this will change in https://github.com/learningequality/kolibri/issues/13398. But as it was already implemented I have left it here for now.